lib/routes/home/widgets/payments_list/dialog/tx_widget.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:breez_translations/breez_translations_locales.dart';
+import 'package:c_breez/services/injector.dart';
+import 'package:c_breez/widgets/flushbar.dart';
+import 'package:c_breez/widgets/link_launcher.dart';
+import 'package:flutter/material.dart';
+
+class TxWidget extends StatelessWidget {
```

Comment:
Does this differ from TxWidget on payment_details_dialog? If not let's remove the TxWidget inside payment_details_dialog.

Reply:
@erdemyerebasmaz I'm not finding which widget you are mentioning. Can you share it here, please?

Reply:
Seems it's no longer or main branch or I've mistaken it for breezmobile code at the time of writing this.